webview/__init__.py

```diff
@@ -60,21 +71,33 @@
 DEFAULT_HTTP_PORT = 42001
 
 guilib = None
-_debug = {
-  'mode': False
-}
+_debug = {'mode': False}
 _user_agent = None
 _http_server = False
 _private_mode = True
 _storage_path = None
 
-token = _token
-windows = []
-menus = []
-
-def start(func=None, args=None, localization={}, gui=None, debug=False, http_server=False,
-          http_port=None, user_agent=None, private_mode=True, storage_path=None, menu=[], 
-          server=http.BottleServer, server_args={}, ssl=False):
+token = _TOKEN
+windows: list[Window] = []
+menus: list[Menu] = []
+
+
+def start(
+    func: Callable[..., None] | None = None,
+    args: Iterable[Any] | None = None,
+    localization: dict[str, str] = {},
+    gui: Literal['cef', 'qt', 'gtk'] | None = None,
```

Comment:
Valid values are ['qt', 'gtk', 'cef', 'mshtml', 'edgechromium']. The comment is outdated.